conf/php.ini

```diff
@@ -1,2 +1,7 @@
-memory_limit = PHP_MEMORY_LIMIT
```

Comment:
Hi @markoshust, I have a question about php.ini

I am not fluent with php but Is there a reason why are we not using the settings suggested by Magento?

https://github.com/magento/magento2/blob/develop/php.ini.sample